my-sms/spec/models/institute_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Institute, :type => :model do
+  subject { institute }
+  let(:institute) { build(:institute, institute_params) }
+  let(:institute_params) { {} }
+
+  context 'with valid params' do
```

Comment:
test validity of the factory

```ruby
{ is_expected.to be_valid }

Reply:
Fixed in cb06b09f0dd87be98de00cf28dce789da850fe1b